pkg/pool-weighted/contracts/WeightedPool.sol

```diff
@@ -141,7 +141,7 @@ contract WeightedPool is IWeightedPool, BalancerPoolToken, PoolInfo, Version {
     }
 
     /// @inheritdoc IBasePool
-    function onSwap(PoolSwapParams memory request) public view virtual onlyVault returns (uint256) {
+    function onSwap(PoolSwapParams memory request) public view virtual returns (uint256) {
```

Comment:
For documentation, maybe we should create a separate PR with this specific change? @jubeira , what do you think

Reply:
OPhhhh, nevermind then, sorry :)